editoast/src/core/mod.rs

```diff
@@ -291,13 +282,6 @@ enum CoreError {
     #[error("Cannot parse Core response: {msg}")]
     #[editoast_error(status = 500)]
     CoreResponseFormatError { msg: String },
-    /// A standard core error was found in the response, so it is forwarded
```

Comment:
Could you add back that variant with `core_error: StandardCoreError`?

Reply:
I don't get what you mean

Reply:
Let's keep it this :sadge: